pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "saleor"
-version = "2.11.0-a.0"
+version = "2.11.0-rc.2"
```

Comment:
These version changes (here and above) are unrelated to this PR so we shouldn't change it.

Reply:
I checked out from 2.11 branch and create merge request for Master?
 
Is it right or i was supposed to do something else?

Reply:
You should check out from `master`. `2.11` is a release branch that we maintain and we're currently using it to prepare the `2.11` release.

Reply:
OK.I will create a new PR.